backend/configs/usr/lib/cgi-bin/download_rootfs.sh

```
@@ -0,0 +1,41 @@
+
+function cleanup()
+{
+    sudo umount -f "$MOUNT_DIR" || true
+    rm -r "$MOUNT_DIR"
+}
+
+trap cleanup EXIT
+
+sudo mount "$ROOTFS_DEV" "$MOUNT_DIR"
+
+if [ -z "$(ls -A $MOUNT_DIR)" ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)